Python/Default.sublime-keymap

```diff
@@ -1,25 +1,110 @@
 [
-    // Auto-pair quotes even after string modifiers.
-    // Copied over from the default bindings with modifications to `preceding_text`
-    // and an added selector condition.
+    // Suppress default auto-pairing bindings to avoid unwanted interference
+    { "keys": ["\""], "command": "insert", "args": {"characters": "\""}, "context":
+        [
+            { "key": "setting.auto_match_enabled" },
+            { "key": "selector", "operand": "source.python" },
```

Comment:
This keybinding also disables the "step over closing quote" functionality from the default keybindings. You would need to exclude the `punctuation.definition.string.end` scope from the selector to prevent this, similar to #3204.